src/lib.rs

```diff
@@ -166,7 +166,7 @@ mod future_result;
 
 // Re-exports
 pub use common::DateTime;
-pub use {columnar, query_grammar, time};
+pub use {columnar, query_grammar, rayon, time};
```

Comment:
It did not miss anything, we actually only need to expose `rayon_core` because we only use `ThreadPool` in our public API and not `ParallelIterator` et al. `rayon_core` is supposedly much more stable exactly to make it easier to share with other crates in the dependency tree.